src/cil.mli

```diff
@@ -1044,6 +1044,7 @@ and instr =
                   (string option * string * exp) list *
                                         (* inputs with optional names and constraints *)
                   string list *         (* register clobbers *)
+                  string list) option * (* goto locations *)
```

Comment:
It's not clear from here, why this additional `option` wrapping is necessary or what the `None` then means.